src/MSBuildLocator/DotNetSdkLocationHelper.cs

```diff
@@ -106,13 +117,44 @@ private static string GetDotNetBasePath(string workingDirectory)
 
             var outputString = string.Join(Environment.NewLine, lines);
 
+            var lineSdkIndex = lines.FindIndex(line => line.Contains(".NET Core SDKs installed"));
+
+            if (lineSdkIndex != -1)
+            {
+                lineSdkIndex++; 
+
+                while (lineSdkIndex < lines.Count && !lines[lineSdkIndex].Contains(".NET Core runtimes installed"))
+                {
+                    var ma = SdkRegex.Match(lines[lineSdkIndex]);
```

Comment:
Use a variable name like `sdkMatch` to provide more meaning.

Reply:
Fixed.